plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -94,46 +95,51 @@ private static Session getTestSession(Identity identity)
     @Test
     public void testCacheRefreshOnGrantAndRevoke()
     {
-        assertThatThrownBy(() -> queryRunner.execute(ALICE, "SELECT * FROM test"))
+        assertThatThrownBy(() -> getQueryRunner().execute(ALICE, "SELECT * FROM test"))
```

Comment:
nit: extract such change into preparatory commit.

Reply:
Split to more granullar operations. 
- use query testing framework
- move to trino-hive
- merge redundant tests